src/App.js

```diff
@@ -1 +1,97 @@
 
+import React from "react";
+import "./App.css";
+import { Button, Card, Form } from 'react-bootstrap';
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+
+function Todo({ todo, index, markTodo, removeTodo }) {
+  return (
+    <div
+      className="todo"
+    >
+      <span style={{ textDecoration: todo.isDone ? "line-through" : "" }}>{todo.text}</span>
+      <div>
+        <Button variant="outline-success" onClick={() => markTodo(index)}>✓</Button>{' '}
```

Comment:
should it be modificator instead?

Reply:
You mean BEM naming modification, right?